generated/nifgen/nifgen/session.py

```diff
@@ -3076,12 +3076,18 @@ def __init__(self, resource_name, channel_name=None, reset_device=False, options
                 | driver_setup            | {}      |
                 +-------------------------+---------+
 
+            _grpc_channel (grpc.Channel): MeasurementLink gRPC channel
+
 
         Returns:
             session (nifgen.Session): A session object representing the device.
 
         '''
-        library_interpreter = _library_interpreter.LibraryInterpreter(encoding='windows-1251')
+        if _grpc_channel:
+            import nifgen._grpc as _grpc
+            library_interpreter = _grpc.GrpcStubInterpreter(_grpc_channel)
```

Comment:
confirm this renaming is happening in a separate PR please

Reply:
The rename PR has already gone in; needs to be merged in to this PR.